doc/usr/expl/history.md

```diff
@@ -0,0 +1,40 @@
+# History
+
+This project was initially developed in 2016 at a French startup company named
+Stupeflix. It took inspiration from the scene graph paradigm of [OSG] for which
+many clumsy in-house patches were maintained, essentially for video playback
+support and lazy resources management.
+
+The video playback logic was the first foundation to be extracted from these
+patches, and landed in a dedicated module named [sxplayer] (standing for
+"Stupeflix Player"), a free (LGPL) video playback library on top of FFmpeg.
+
+While it helps with maintenance, most of the OSG features remained unneeded,
+the Python bindings were painful, and the complexity of the stack of patches
+added on top was awkward to manage. It was decided to prototype a new engine,
```

Comment:
We should mention that the OSG version used at the time was frozen and was an already outdated version for the time. We were unable to rebase the whole stack of patches/hacks (including the custom python binding) back to upstream.

Reply:
Reworked that paragraph to add this information